pygaggle/qa/__main__.py

```diff
@@ -17,6 +18,15 @@ def define_retriever_args(parser):
     parser.add_argument('--corpus', type=str, required=True, help="Pyserini sparse index name or path, serve as corpus")
 
 
+def define_type_args(parser):
+    parser.add_argument('qatype', type=str, choices=["openbook", "closedbook"], help="Open-book or closed-book question answering.")
```

Comment:
Should this be --qatype

Reply:
I was thinking positional argument makes commands more concise.

Reply:
We rarely end up using positional arguments in pygaggle so might be better to stick to keyword ones?

Reply:
Can we make openbook qa the default? Also make sure the other instructions for openbookqa still run?